packages/client/modules/teamDashboard/containers/TeamIntegrationsRoot/TeamIntegrationsRoot.tsx

```diff
@@ -1,3 +1,4 @@
+import styled from '@emotion/styled'
```

Comment:
We don't import styled anymore and using tailwind for new components

Reply:
Yes, I just moved this from `TeamIntegrationsWrapper`.